Python/Tests/PythonToolsMockTests/NavigableTests.cs

```diff
@@ -56,7 +56,7 @@ public class NavigableTests {
 ";
             using (var helper = new NavigableHelper(code, Version)) {
                 // os
-                await helper.CheckDefinitionLocations(7, 2, ExternalLocation(1, 1, "os.py"), Location(1, 8));
```

Comment:
`os.py` opens fine, but there is no second location.  Current `FindReferences` only finds location in the `ImportedModuleNameWalker`. May be related to the `ModuleImportDefinition` being broken.